app/src/main/java/ch/hikemate/app/ui/saved/SavedHikesScreen.kt

```diff
@@ -62,23 +60,27 @@ object SavedHikesScreen {
 }
 
 @Composable
-fun SavedHikesScreen(
-    savedHikesViewModel: SavedHikesViewModel = viewModel(factory = SavedHikesViewModel.Factory),
-    navigationActions: NavigationActions
-) {
+fun SavedHikesScreen(hikesViewModel: HikesViewModel, navigationActions: NavigationActions) {
```

Comment:
Warning: You have a sonar issue here. Please correct it

Reply:
![image](https://github.com/user-attachments/assets/7cc4ec7d-bacb-4c79-abfa-4016931fdd55)

This is a personal best